testnets/xiontestnet/assetlist.json

```diff
@@ -0,0 +1,138 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "xion-testnet-1",

```

Comment:
please change chain_name to 'xiontestnet' in this file and the chain.json file. (xion-test-1 will still be recorded as chain_id)

Reply:
Updated `chain_name` to `xiontestnet`. Also renamed in chain cfg. Thanks!